content/docs/07-reference/ai-sdk-ui/03-use-object.mdx

```diff
@@ -65,21 +65,28 @@ export default function Page() {
     {
       name: 'initialValue?',
       type: 'DeepPartial<RESULT> | undefined',
-      description: 'An optional value for the initial object.',
+      description: 'An value for the initial object. Optional.',
```

Comment:
`optional: true`

Reply:
what does `optional: true` do btw? i did not observe any difference when i set it